content/docs/concepts/traffic-management/network/index.md

```diff
@@ -0,0 +1,325 @@
+{{< /text >}}
+
+See the [circuit-breaking task](/docs/tasks/traffic-management/circuit-breaking/)
+for detailed instructions on how to configure a circuit breaker pattern.
+
+## Health checking
+
+Istio currently supports passive health checking to minimize the chances of
+accessing an unhealthy instance in the load balancing pool. The Envoy proxies
+support active health checking but Istio doesn’t enable it. When you combine
+Istio health checks with platform-level health checks, such as those supported
+by Kubernetes, your applications can quickly eject unhealthy pods, containers,
+or VMs from the service mesh, minimizing request failures and impact on
+latency.
+
+The Envoy proxies follow a circuit breaker pattern to classify instances as
+unhealthy or healthy based on their failure rates for the health check API
+call. The process to remove unhealthy instances from the load balancing pool is
+independent of platform:
+
+1. Envoy periodically checks the health of each instance in the load balancing
+   pool.
+
+1. When the number of failed health checks for a given instance exceeds a
+   specified threshold, the Envoy proxy removes the instance from the service's
+   load balancing pool.
+
+1. When the number of passed health checks exceeds a specified threshold, the
+   Envoy proxy adds the instance back into the service's load balancing pool.
+
+The following diagram shows steps 1-3 in the process of removing an unhealthy
+instance and restoring it when it is healthy:
+
+{{< image width="40%"
+    link="./network-1.svg"
+    caption="Health checking process"
+    >}}
+
+Services can shed loads by returning a Service Unavailable (`HTTP 503`)
+response to a health check. The Envoy proxy for the calling service immediately
+removes the service instance from its load balancing pool.
+
+## Fault injection
+
+You can use fault injection to test the end-to-end failure recovery capability
+of the application as a whole. An incorrect configuration of the failure
+recovery policies could result in unavailability of critical services. Examples
+of incorrect configurations include incompatible or restrictive timeouts across
+service calls.
+
+With Istio, you can use application-layer fault injection instead of killing
+pods, delaying packets, or corrupting packets at the TCP layer. You can inject
+more relevant failures at the application layer, such as HTTP error codes, to
+test the resilience of an application.
+
+You can inject faults into requests that match specific conditions, and you can
+restrict the percentage of requests Istio subjects to faults.
+
+You can inject two types of faults:
+
+-  **Delays:** Delays are timing failures. They mimic increased network latency
+   or an overloaded upstream service.
+
+-  **Aborts:** Aborts are crash failures. They mimic failures in upstream
+   services. Aborts usually manifest in the form of HTTP error codes or TCP
+   connection failures.
+
+### Introduce a 5 second delay in 10% of requests
```

Comment:
This heading should come after the following paragraph which talks about both delays and aborts